.github/workflows/build.yml

```diff
@@ -474,11 +511,11 @@ jobs:
           echo "" >> comment.md
 
           # ls -alh
-          # ls -alh app/build/outputs/connected_android_test_additional_output/debugAndroidTest/connected/
-          # stat 'app/build/outputs/connected_android_test_additional_output/debugAndroidTest/connected/Pixel_2(AVD) - 11/InitActivityTest_saveDeviceScreenBitmap.png'
+          # ls -alh app/build/outputs/connected_android_test_additional_output/gvrDebugAndroidTest/connected/
+          # stat 'app/build/outputs/connected_android_test_additional_output/gvrDebugAndroidTest/connected/Pixel_2(AVD) - 11/InitActivityTest_saveDeviceScreenBitmap.png'
 
           # for each AVD Image
-          for ss_test_avd in app/build/outputs/connected_android_test_additional_output/debugAndroidTest/connected/*; do 
+          for ss_test_avd in app/build/outputs/connected_android_test_additional_output/gvrDebugAndroidTest/connected/*; do 
```

Comment:
Are there no ss_tests for noGvr flavour ? Or did i miss them somwhere ?

Reply:
They are generated, but not uploaded to github as I wanted to have passing test first

Reply:
Done